src/vs/platform/editor/common/editor.ts

```diff
@@ -320,6 +325,11 @@ export interface IEditorOptions {
 		 */
 		alwaysOnTop?: boolean;
 	};
+
+	/**
+	 * A hint of a possible title for the input
+	 */
+	readonly preferredTitle?: string;
```

Comment:
I had to add this to get the types to line up here:

https://github.com/microsoft/vscode/pull/259108/files#diff-627f916fdc6850576b593ff6a39ec097231faa80387f331edfb8c4c147bb7c2cR915

Reply:
I don't understand why you need it on this interface?